### ISSUE TITLE
Cosmos SDK Liquidity Module docs review/QA

### ISSUE THREAD

**Issue by SenorMonito** [tendermint/liquidity#171]
The aim of the epic is to QA the documentation artifacts that form part of the Liquidity module. This includes the following

- PRIMARY FOCUS: [Liquidity Module Spec Markdown docs](https://github.com/tendermint/liquidity/blob/develop/x/liquidity/spec)
   ongoing in https://github.com/tendermint/liquidity/pull/201 
- [Swagger HTTP API doc](https://app.swaggerhub.com/apis-docs/bharvest/cosmos-sdk_liquidity_module_rest_and_g_rpc_gateway_docs) (2.1.1)
   review cycle complete in https://github.com/tendermint/liquidity/pull/214
- [Tendermint published godoc](https://pkg.go.dev/github.com/tendermint/liquidity) 
- [Liquidityd Client CLI doc Markdown](https://github.com/tendermint/liquidity/blob/develop/doc/client.md) 
   first review cycle complete in go files, PR https://github.com/tendermint/liquidity/pull/212

Note, Litepaper has been QA separately.
- [Liquidity Module Lite Paper (English) source](https://github.com/SenorMonito/LiqLitePaper) and [PDF](https://github.com/tendermint/liquidity/blob/develop/doc/LiquidityModuleLightPaper_EN.pdf)

Issues are raised for each documentation area

Outcomes of each issue will include

- Edits suggested via PR
- Comments that will outline areas for improvements and identify
- It may also include plans for remediation and how to write better documentation

**Comment by barriebyron** [comment]
to help with EcoDev planning, can you let @tosch110 and me know what the target due date is?

**Comment by barriebyron** [comment]
Status on Friday April 9

is pool-type-id flag the same thing as pool-id? I see an opportunity for clarity. 

**Liquidity Module Spec Markdown docs**

- [#201](https://app.zenhub.com/workspaces/gravity-dex-603e5e9d98bebc000fea8313/issues/tendermint/liquidity/171) 

- Extensive first-round and second-round editorial work complete. I still lack a fundamental understanding of how cool the liquidity pool module is, forgive my "I'm still learning" state. 

- Several outstanding questions remain open in the Files Changes view.

**Liquidity Pool liquidityd command-line help Client CLI Documentation QA**

- [#212](https://github.com/tendermint/liquidity/pull/212) 

- First and second rounds complete. Excellent progress getting each flag explained, see [this comment](https://github.com/tendermint/liquidity/pull/212/files#diff-3d2dbc99856e2b9dd214922be356cf8e80279c9a6d411e38a149f600d2bc346eR45

**Swagger**

- [#206](https://github.com/tendermint/liquidity/pull/206) is in draft status. Some of the required changes look so familiar, I am wondering if we shouldn't merge and open a new PR. Backstory, the original PR was merged before I looked at every detail. Let's open a new PR if needed. I see repetitive changes that we can iterate on. 

**godoc**

- [#217](https://app.zenhub.com/workspaces/gravity-dex-603e5e9d98bebc000fea8313/issues/tendermint/liquidity/217) remind me what a godoc is and how we expose it? purpose/audience? Next up!

**Comment by barriebyron** [comment]
Now that the liquidity module is ready to be merged, here's a summary of where we are with the docs work:

https://github.com/tendermint/liquidity/pull/201 ready for final review

- just one broken link for SwaggerHub API docs needs attention, maybe @kogisin can address?
- a good basis for future iterations 
- README edits are incomplete, let's do this:
   - merge this PR and then get the README text changes from https://github.com/tendermint/liquidity/pull/234/files/d70413c44113c04a97c1ebc8035852529b0527f1#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5  and the README CLI changes from the generated code (or client.md file, whatever works on your end)

I realize we have a lot of loose ends, so to expedite the liquidity module merged, we can always agree to iterate and improve. 

Questions? please ask. I'm open to ideas and suggestions (let's not do this long running, wide scope PR again, let's use smaller PRs with more manageable content reviews)... we are all learning here. Thank you for your patience. 
@SenorMonito FYI for status

**Comment by kogisin** [comment]
Thanks for the status. I will update the broken link for SwagerHub API docs today. 

> let's use smaller PRs with more manageable content reviews)...

This will certainly speed up the process.
